src/components/MovieForm/MovieForm.tsx

```diff
@@ -0,0 +1,152 @@
+                       type="date"
+                       id="releaseDate"
+                       name="releaseDate"
+                       aria-label="Release Date"
+                       defaultValue={initialMovieInfo?.releaseDate || ""}
+                       placeholder="Select Date"
+                       required={true}
+                />
+            </div>
+
+            <div className={styles.labelAndInputContainer}>
+                <label className={styles.inputLabel}
+                       htmlFor="movieUrl">Movie URL</label>
+                <input className={styles.input}
+                       type="url"
+                       id="movieUrl"
+                       name="movieUrl"
+                       aria-label="Movie URL"
+                       defaultValue={initialMovieInfo?.movieUrl || ""}
+                       placeholder="https://"
+                       required={true}
+                />
+            </div>
+
+            <div className={styles.labelAndInputContainer}>
+                <label className={styles.inputLabel}
+                       htmlFor="rating">Rating</label>
+                <input className={styles.input}
+                       type="number"
+                       id="rating"
+                       name="rating"
+                       aria-label="Movie Rating"
+                       defaultValue={initialMovieInfo?.rating || ""}
+                       placeholder="7.8"
+                       required={true}
+                />
+            </div>
+
+            <div className={styles.labelAndInputContainer}>
+                <label className={styles.inputLabel}
+                       htmlFor="genres">Genre</label>
+                <select className={styles.input}
+                        id="genres"
+                        name="genres"
+                        aria-label="Movie Genres"
+                        defaultValue={initialMovieInfo?.genres || [""]}
+                        multiple={true}
+                        required={true}
+                >
+                    <option value="Action">Action</option>
+                    <option value="Crime">Crime</option>
+                    <option value="Documentary">Documentary</option>
+                    <option value="Thriller">Thriller</option>
+                    <option value="Musical">Musical</option>
+                </select>
+            </div>
+
+            <div className={styles.labelAndInputContainer}>
+                <label className={styles.inputLabel}
+                       htmlFor="runtime">Runtime</label>
+                <input className={styles.input}
+                       type="text"
+                       id="runtime"
+                       name="runtime"
+                       aria-label="Movie Runtime"
+                       defaultValue={initialMovieInfo?.runtime || ""}
+                       placeholder="minutes"
+                       required={true}
```

Comment:
let's try to create some component probably for inputs and reuse it. 
![image](https://github.com/user-attachments/assets/5907acd8-bb01-41e1-b789-c3d8b63ed1c2)

because there are a lot of repeted code. You can create component, pass props and reuse it. It can be not for all fields, but if it will be for 3-4 of them we decrease the code in a good count